test/runtests.jl

```diff
@@ -2,8 +2,17 @@ using DataFrames
 using IPUMS
 using Test
 
+
 @testset "Helpers" begin
     res = ipums_data_collections()
     @test DataFrame == typeof(res)
     @test (14, 4) == size(res)
 end
+
+@testset "DDI Parser" begin
+    ddi = parse_ddi("testdata/cps_00157.xml")
+    @test ddi.extract_date == "2023-07-10"
+    @test ddi.variable_info[1].position_end == 4
```

Comment:
Since this seems like the core piece of the parser (or at least one of the major components), could you add a test for `ddi._ns`?

Reply:
I added the test case for `ddi._ns`.